### PR TITLE
Fix emulate services and change adb

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

File: mobile/android/android-emulator-services.ts [hunk 1]
```diff
@@ -27,6 +27,11 @@ class AndroidEmulatorServices implements Mobile.IEmulatorPlatformServices {
 	private static UNABLE_TO_START_EMULATOR_MESSAGE = "Cannot run your app in the native emulator. Increase the timeout of the operation with the --timeout option or try to restart your adb server with 'adb kill-server' command. Alternatively, run the Android Virtual Device manager and increase the allocated RAM for the virtual device.";
 	private static RUNNING_ANDROID_EMULATOR_REGEX = /^(emulator-\d+)\s+device$/;
 
+	private static MISSING_SDK_MESSAGE = "The Android SDK is not configured properly. " +
```
Comment (teobugslayer) [inline mobile/android/android-emulator-services.ts]: This string is not used in the new code, is it needed?

Comment (Fatme) [inline mobile/android/android-emulator-services.ts]: Yes, we use it `checkAndroidSDKConfiguration` function

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by Fatme** [comment]
It should be reopened against release!

**Pull request closed without merging**